package.json

```diff
@@ -44,7 +44,7 @@
     "execa": "^4.0.2",
     "json-formatter-js": "^1.3.0",
     "kuzzle-sdk-v6": "npm:kuzzle-sdk@^6.2.7",
-    "kuzzle-sdk-v7": "npm:kuzzle-sdk@^7.4.0",
+    "kuzzle-sdk-v7": "npm:kuzzle-sdk@7.4.7",
```

Comment:
:eyes:

Reply:
It is to fix sdk version until we merge  [sdk js hotfix](https://github.com/kuzzleio/sdk-javascript/pull/589)
there are build errors in AC using sdk v7.4.9